slash-commands/moderation/ban.js

```diff
@@ -0,0 +1,111 @@
+const Discord = require('discord.js');
+const {SlashCommandBuilder} = require('@discordjs/builders');
+const {promisePool} = require('../../config/db');
+const {sendToAuditLogsChannel} = require('../../helpers/sendToAuditLogs');
+const {
+  isServerStaff,
+  sendNoTargetStaffReply,
+} = require('../../helpers/validation');
+const {verifyReasonLength} = require('../../helpers/stringHelpers');
+
+const banIntro = "You've been banned for the following reason: ```";
+const unbanRequest =
+  ' ``` If you wish to challenge this ban, please submit a response in this Google Form: https://forms.gle/KxTMhPbi866r2FEz5';
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('ban')
+    .setDefaultPermission(false)
+    .setDescription('Bans a user')
+    .addUserOption((option) =>
+      option.setName('target').setDescription('The user').setRequired(true)
+    )
+    .addStringOption((option) =>
+      option.setName('reason').setDescription('The reason').setRequired(true)
+    )
+    .addIntegerOption((option) =>
+      option
+        .setName('days')
+        .setDescription(
+          'Number of days of messages to delete (between 0 and 7)'
```

Comment:
Should specify that it's between 0 and 7, inclusive.

Reply:
I made this change

Reply:
I added a dropdown menu instead which will make it obvious that the options are from 0 to 7.